http/grpc/server.go

```diff
@@ -0,0 +1,55 @@
+package grpc
+
+import (
+	"context"
+	"fmt"
+	"raccoon/config"
+	"raccoon/metrics"
+	"raccoon/pkg/collection"
+	"raccoon/pkg/identification"
```

Comment:
We should not have common package names like `model`, it's not idiomatic. Ideally, package name should describe what it contains. One interesting [read ](https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common) regarding this.

Reply:
Do you see `model` not describing what it composes of ?

Interesting share. However not sure if it's relevant here though. The author talks about using names like `base`, `common`, `helpers` etc., which I agree contextually (if they are not reusable elsewhere). 
```
[A little] duplication is far cheaper than the wrong abstraction.

— Sandy Metz
```
is debatable. 

`model` works here because it packages all kind of DTOs in 1 place and it imports this `model`  into multiple handlers. If at all, we need to change/introduce a new field into `id {}`, it's easily done in 1 place!